acmwebsite/model/banner.py

```diff
@@ -0,0 +1,13 @@
+from acmwebsite.model import DeclarativeBase
+from sqlalchemy import Column
+from sqlalchemy.types import Integer, Unicode
+from depot.fields.sqlalchemy import UploadedFileField
+from depot.fields.specialized.image import UploadedImageWithThumb
+
+
+class Banner(DeclarativeBase):
+    __tablename__ = 'banner'
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    photo = Column(UploadedFileField(upload_type=UploadedImageWithThumb))
+    description = Column(Unicode(255), unique=True)
```

Comment:
@jackrosenthal, what DB is backing this? Postgresql?

If so, @edwargix, use `Text` instead of `Unicode(255)` for this field, we don't want to limit descriptions to 255 chars.

Reply:
Yes, although it is good to keep a limit on the size (important for some other databases, MySQL cough cough, and I don't want to break compatibility for this one thing).

255 bytes is a bit small tho

Reply:
@jackrosenthal , proposals for a more appropriate size? I'm thinking 512-1000 is fine.